src/fieldAnalyser/inconsistencies.ts

```diff
@@ -1,6 +1,6 @@
 import {FieldTypes} from '@coveord/platform-client';
 
-export class Inconsistencies extends Map {
+export class Inconsistencies extends Map<string, Set<FieldTypes>> {
```

Comment:
Does that compile? Pretty sure TypeScript `(╯‵□′)╯︵┻━┻` when I tried

Reply:
yes it worked.